storage/Devices/FilesystemImpl.cc

```diff
@@ -396,14 +396,26 @@ namespace storage
     Text
     Filesystem::Impl::do_umount_text(const string& mountpoint, bool doing) const
     {
-	return sformat(_("Unmount %1$s"), get_displayname().c_str());
+	const BlkDevice* blkdevice = get_blkdevice();
+
+	return sformat(_("Unmount %1$s at %2$s"), blkdevice->get_name().c_str(),
+		       mountpoint.c_str());
```

Comment:
That parameter will be used to distinguish whether the text is used in the action overview ("Unmount ...") or during committing ("Unmounting ...") (just like in the old libstorage).